docs/src/forc/commands/forc_parse-bytecode.md

```diff
@@ -19,23 +19,80 @@ Print help information
 
 ## EXAMPLES:
 
-Example with the initial project created using `forc init`:
+We can try this command with the initial project created using `forc init`, with the counter template:
 
-```console
-$ forc build -o obj
-Compiled script "my-fuel-project".
-Bytecode size is 28 bytes.
+```sh
+forc init --template counter counter
+cd counter
+forc build -o obj
 ```
 
 ```console
-my-second-project$ forc parse-bytecode obj
-
- half-word  byte  op               raw                notes
-         0  0     JI(4)            [144, 0, 0, 4]     conditionally jumps to byte 16
-         1  4     NOOP             [71, 0, 0, 0]
-         2  8     Undefined        [0, 0, 0, 0]       data section offset lo (0)
-         3  12    Undefined        [0, 0, 0, 28]      data section offset hi (28)
-         4  16    LW(46, 12, 1)    [93, 184, 192, 1]
-         5  20    ADD(46, 46, 12)  [16, 186, 227, 0]
-         6  24    RET(0)           [36, 0, 0, 0]
+counter$ forc parse-bytecode obj
+
+  half-word   byte   op                   raw           notes
+          0   0      JI(4)                90 00 00 04   conditionally jumps to byte 16
+          1   4      NOOP                 47 00 00 00
+          2   8      Undefined            00 00 00 00   data section offset lo (0)
+          3   12     Undefined            00 00 00 c8   data section offset hi (200)
+          4   16     LW(63, 12, 1)        5d fc c0 01
+          5   20     ADD(63, 63, 12)      10 ff f3 00
+          6   24     LW(17, 6, 73)        5d 44 60 49
```

Comment:
Since these will be examples in the book, you can shorten this by adding `// --snip--` here

Reply:
Is this utility supported natively? Doesn't seem to work, unless we have to add [this](https://lib.rs/crates/mdbook-snips)?

Reply:
Ah, you know what, this is my mistake. I was thinking in Rust's documentation that translates to md, and this is just pure md. Disregard my previous comments unless you've found some other reasonable solution, I think this is fine!

Reply:
For reference: https://doc.rust-lang.org/rustdoc/what-to-include.html#what-to-exclude

Reply:
If using the `snip` works with that crate, that would be much cleaner

Reply:
I realized that crate doesn't work either because it only works on `rust` code blocks. Perhaps I can simply manually truncate the example output in the middle and show something like this?

```
counter$ forc parse-bytecode obj

  half-word   byte   op                   raw           notes
          0   0      JI(4)                90 00 00 04   conditionally jumps to byte 16
          1   4      NOOP                 47 00 00 00
          2   8      Undefined            00 00 00 00   data section offset lo (0)
          3   12     Undefined            00 00 00 c8   data section offset hi (200)
          4   16     LW(63, 12, 1)        5d fc c0 01
          5   20     ADD(63, 63, 12)      10 ff f3 00
          ...
          ...
         61   244    Undefined            fa f9 0d d3
         62   248    Undefined            00 00 00 00
         63   252    Undefined            00 00 00 c8

```

Reply:
If you can't figure out how to snip after a bit of exploring, you can always defer it to a future PR. Just file an issue.

Reply:
Yep - I think I will just simply manually truncate it for now to shorten the example output